containeranalysis/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.2.0
+
+07-12-2019 16:56 PDT
+
+
+### Implementation Changes
+- Add 'options_' argument to clients' 'get_iam_policy'; pin black version (via synth). ([#8653](https://github.com/googleapis/google-cloud-python/pull/8653))
```

Comment:
The first part is likely a new feature.  I don't know whether we should duplicate / split the entry to have the `black` part show up down in "Internal / Testing Changes".